src/renderer/components/playlist-info/playlist-info.vue

```diff
@@ -68,11 +98,71 @@
         </h3>
       </div>
 
-      <ft-share-button
-        v-if="!hideSharingActions"
-        :id="id"
-        :dropdown-position-y="description ? 'top' : 'bottom'"
-        share-target-type="Playlist"
+      <div class="playlistOptions">
+        <ft-icon-button
+          v-if="editMode"
+          :title="$t('User Playlists.Save Changes')"
+          :icon="['fas', 'save']"
+          theme="secondary"
+          @click="savePlaylistInfo"
+        />
+        <ft-icon-button
+          v-if="editMode"
+          :title="$t('User Playlists.Cancel')"
+          :icon="['fas', 'times']"
+          theme="secondary"
+          @click="exitEditMode"
+        />
+
+        <ft-icon-button
+          v-if="!editMode && infoSource === 'user'"
+          :title="$t('User Playlists.Edit Playlist Info')"
+          :icon="['fas', 'edit']"
+          theme="secondary"
+          @click="enterEditMode"
+        />
+        <ft-icon-button
+          v-if="videoCount > 0 && showPlaylists && !editMode"
+          :title="$t('User Playlists.Copy Playlist')"
+          :icon="['fas', 'copy']"
+          theme="secondary"
+          @click="toggleCopyVideosPrompt"
+        />
+        <ft-icon-button
+          v-if="!editMode && infoSource === 'user' && videoCount > 0"
+          :title="$t('User Playlists.Remove Watched Videos')"
+          :icon="['fas', 'eye-slash']"
```

Comment:
**issue (non-blocking):** The purpose of this button is incredibly unclear by looking at the icon. This should be less of an issue once we have icon labels, but please see if you can find a better one.

**issue (non-blocking):** On top of the previous issue, this is a destructive action, which we should have clearly marked. In the design world, destructive controls are colored red to clearly communicate to the user that they have a lasting destructive impact. The prompt will hopefully stop most people from making a mistake, but it's still a notable issue. My suggestion is to Implement a color override property on the `ft-icon-button` component and use enforce a red color here (e.g., the red primary color variable and the primary-text-color for that).

Reply:
https://fontawesome.com/search?q=slash&o=r&m=free 
Not that many icons have slash, help me find a better icon I guess ._.

Reply:
Color is updated, I kind of waiting for any of you to comment on it

Reply:
Yeah, I don't see anything else better in the `fontawesome` list by itself ([without using layers, at least](https://origin.fontawesome.com/docs/web/style/layer)), so that's fine.

Reply:
The color is updated to use the primary color, not red, which is different than what I was intending. Main point is that forcing red in particular endows it with special meaning (for everyone else except someone with a `Secondary Color Theme` of `Red`, I guess, which you can mitigate by using a darker red.

Reply:
I am following https://github.com/FreeTubeApp/FreeTube/blob/development/src/renderer/components/privacy-settings/privacy-settings.vue#L43-L62
I think it's better to have later PRs to do this kind of minor changes which probably also affects other places

Reply:
Sounds good, I've placed this into its own issue